troubadix/plugins/script_xref_url.py

```diff
@@ -63,11 +63,65 @@ def check_content(
         )
         for match in matches:
             if match:
-                if match.group("value") not in ALLOWED_URLS and not url(
-                    match.group("value"), strict_query=False
-                ):
-                    yield LinterError(
-                        f"{match.group(0)}: Invalid URL value",
-                        file=nasl_file,
-                        plugin=self.name,
-                    )
+                url_value = match.group("value")
+                full_xref = match.group(0)
+                if url_value not in ALLOWED_URLS:
+
+                    # First a few possible malformed URLs introduced by e.g. some automatic
+                    # extraction or by copy'n'paste
+                    if url_value.endswith(">") and "<" not in url_value:
```

Comment:
```suggestion
                if url_value in ALLOWED_URLS:
                    continue

                # First a few possible malformed URLs introduced by e.g. some automatic
                # extraction or by copy'n'paste
                if url_value.endswith(">") and "<" not in url_value:
```
Now that we have a larger block of code in here, we should return early to keep the indentation low (only included the first few lines in my suggestion to keep the suggestion small)

Reply:
Ah, nice catch / hint and indeed makes sense. I have also moved the `full_xref = match.group(0)` after the first continue to only assign this if really required.